### DISCUSSION TITLE
Show timezone of cron in Airflow main screen

### DISCUSSION THREAD

**Discussion by miky-schreiber** [apache/airflow#27949] [Ideas] [answered]
In our organization, there are many DAGs which are scheduled by different timezones to meet different customers' use-cases. The main Airflow screen shows the cron expressions without timezones. It would be great to show an indication of the timezone.

**Comment by potiuk** [comment, marked as answer]
Sure. PR about it would be cool. Also you can create a small feature for that - we can mark it as "good-first-issue" and somone might pick it up - if you or your organisation does not feel like contributing such feature directly as PR.
